kodi_addon_checker/__main__.py

```diff
@@ -39,15 +56,28 @@ def main():
                                      the current directory.")
     parser.add_argument("--version", action="version",
                         version="%(prog)s 0.0.1")
-    parser.add_argument("add_on", metavar="add-on", type=dir_type, nargs="*",
-                        help="optional add-on directories")
+    parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
     ConfigManager.fill_cmd_args(parser)
     args = parser.parse_args()
 
-    repo_path = os.path.abspath(os.getcwd())
-    config = Config(repo_path, args)
-    ConfigManager.process_config(config)
-    check_repo(config, repo_path, args.add_on)
+    if args.dir:
+        # Following report is a wrapper for all sub reports
+        report = Report("")
+        for directory in args.dir:
+            report.add(check_artifact(directory, args))
+    else:
+        report = check_artifact(os.getcwd(), args)
+
+    if report.problem_count > 0:
+        report.add(Record(PROBLEM, "We found %s problems and %s warnings, please check the logfile." %
+                          (report.problem_count, report.warning_count)))
+    elif report.warning_count > 0:
+        report.add(Record(WARNING, "We found %s problems and %s warnings, please check the logfile." %
+                          (report.problem_count, report.warning_count)))
```

Comment:
`problem_count` will be 0 in this case, so maybe log `We found no problems and ...` to be more consistent with the output for no problems & warnings.

Reply:
Fixed with https://github.com/xbmc/addon-check/pull/67/commits/03dfd4372a645855cdb2d8677c78547ec5bfbae8